spec/support/shared_examples/address_balances.rb

```diff
@@ -129,6 +129,30 @@
       end
     end
   end
+
+  describe '#transactions' do
+    let(:transaction) { build(:transaction_model, from_address_id: 'from_address') }
+    let(:response) do
+      Coinbase::Client::AddressTransactionList.new(
+        data: [
+          transaction
+        ]
+      )
+    end
+
+    before do
+      allow(external_addresses_api)
+        .to receive(:list_address_transactions)
+        .with(normalized_network_id, address_id, { limit: 100, page: nil })
+        .and_return(response)
+    end
+
+    context 'when list transactions with correct input' do
+      it 'returns the correct transactions' do
+        expect(address.transactions.first.from_address_id).to eq 'from_address'
+      end
```

Comment:
We have a shared example for paginated enumerators, so we can just set this whole test up, very similarly to:

https://github.com/coinbase/coinbase-sdk-ruby/blob/81b50ed50158d57cb1305b694cf412895b3d0bf2/spec/unit/coinbase/transfer_spec.rb#L97-L110

Reply:
Gracias la verdad seré muy sincero tengo bastante tiempo de querer comunicarme porque he notado cosas muy raras muchas veces an llegado correos sin contenido y otras cosas que he visto y Gálvez por mi diosa yo no he podido comunicarme también tengo como 2 meses que no he podido entra a mi cuenta si me pueden ayudar y por lo mismo cánsela ron por un tiempo mi cuenta bancaria perdí asta mi trabajo porque no se pudo cancelar mi Aseguranza de trabajo por que tenía las cuentas cancelada devido a esta situación y tengo pruebas